recipe/meta.yaml

```diff
@@ -19,11 +19,14 @@ build:
     - jupyter-troubleshoot = jupyter_core.troubleshoot:main
 
 requirements:
+  build:
+    - python
   host:
     - pip
     - python
     - setuptools >=60.0
```

Comment:
As we use `hatchling` as a build backend we can remove `setuptools`, see https://github.com/jupyter/jupyter_core/blob/d3f61f336bfe5c6c63e1af0d8c9fb0dea59f8793/pyproject.toml#L3 and https://github.com/jupyter/jupyter_core/pull/265

Reply:
eliminate unnecesary dependencies